src/Admin/Models/OrganizationEditModel.cs

```diff
@@ -6,13 +6,20 @@
 using Bit.Core.Models.Data.Organizations.OrganizationUsers;
 using Bit.Core.Settings;
 using Bit.Core.Utilities;
+using Bit.SharedWeb.Utilities;
 
 namespace Bit.Admin.Models;
 
 public class OrganizationEditModel : OrganizationViewModel
 {
     public OrganizationEditModel() { }
 
+    public OrganizationEditModel(Provider provider)
+    {
+        Provider = provider;
+        Plan = Core.Enums.PlanType.TeamsMonthly.GetDisplayAttribute()?.GetName();
```

Comment:
This is only setting the `Plan` name, what about the `PlanType`?

Reply:
`PlanType` is already being set by the first item in the select list [here](https://github.com/bitwarden/server/pull/2707/files#diff-8f17f3cfffeea693f9b2a636e6f2a3d9f24abe0f32b81d38b026ff8643dcbe1aR62-R71).

Reply:
I think so - it's important that they're both kept in sync, and there's no guarantee it'll always be the first item on the select list.